### PR TITLE
Always uniq index lists

### PR DESCRIPTION
**Pull request by berggren**
This PR adds safeguards to make sure that the index list is always uniq in order to not balloon the ES HTTP calls.
I have tested a sketch with 500+ timelines without issues.

**Closing issues**
Fixes: #1687

### CODE REVIEW

File: timesketch/api/v1/resources/sketch.py [hunk 1]
```diff
@@ -361,39 +364,34 @@ def get(self, sketch_id):
...
 
+        # Get number of events per timeline
         if sketch_indices:
-            # Stats for index. Num docs per shard.
-            for timeline in sketch.active_timelines:
-                index_name = timeline.searchindex.index_name
+            for index_name in sketch_indices:
                 if indices_metadata[index_name].get('is_legacy', False):
                     doc_count, _ = self.datastore.count(indices=index_name)
                     stats_per_timeline[timeline.id] = {'count': doc_count}
```
Comment (kiddinn) [inline timesketch/api/v1/resources/sketch.py:372]: `timeline.id` isn't defined here... that is the `timeline` isn't defined.

**Review by kiddinn — COMMENTED** [review]
early

File: timesketch/api/v1/resources/sketch.py (hunk 1 above)
Comment (berggren) [reply timesketch/api/v1/resources/sketch.py:373]: Yeah, that is what I fixed before removing the draft status :)

**Review by kiddinn — APPROVED** [review]
LGTM

**Pull request merged**